### PR TITLE
Rs efficiancy

### PR DESCRIPTION
**Pull request by RohanShrothrium**

### CODE REVIEW

File: examples/v2/findBestPath.ts [hunk 1]
```diff
@@ -17,8 +17,8 @@ const amount = parseFloat(args[0]);
     try {
```
Comment (RohanShrothrium) [inline examples/v2/findBestPath.ts:17]: rollback

File: examples/v2/marketBuy.ts [hunk 2]
```diff
@@ -13,19 +13,28 @@ const minAmountOut = parseFloat(args[1]);
...
+        
         const marketParams = await KuruSdk.ParamFetcher.getMarketParams(
             provider,
             contractAddress
         );
         const receipt = await KuruSdk.IOC.placeMarket(signer, contractAddress, marketParams, {
             approveTokens: true,
             size,
-            isBuy: true,
+            isBuy: false,
```
Comment (RohanShrothrium) [inline examples/v2/marketBuy.ts:28]: rollback this line

File: examples/v2/cancelOrder.ts [hunk 3]
```diff
@@ -10,13 +10,19 @@ const args = process.argv.slice(2);
...
 	const provider = new ethers.providers.JsonRpcProvider(rpcUrl);
+    provider._pollingInterval = 100;
     const signer = new ethers.Wallet(privateKey, provider);
 
     try {
         const txReceipt = await KuruSdk.OrderCanceler.cancelOrders(
             signer,
             contractAddress,
-            args.map(arg => BigNumber.from(parseInt(arg)))
+            args.map(arg => BigNumber.from(parseInt(arg))),
```
Comment (RohanShrothrium) [inline examples/v2/cancelOrder.ts:20]: good to implement 80k + 20k*(orderIds - 1) for gas limit logic.

File: src/market/ioc.ts [hunk 4]
```diff
@@ -131,8 +133,10 @@ async function placeAndExecuteMarketBuy(
     quoteSize: number,
     minAmountOut: number,
     isMargin: boolean,
-    isFillOrKill: boolean
+    isFillOrKill: boolean,
+    txOptions?: TransactionOptions
 ): Promise<ContractReceipt> {
+    console.time('Total Market Buy Time');
```
Comment (devblixt) [inline src/market/ioc.ts:139]: should we remove these latency checks?

**Pull request merged**